web/source/keymanweb.js

```diff
@@ -3060,8 +3072,7 @@
   **/
   keymanweb.setDfltKeyboard=function()
   {
-    keymanweb._SetActiveKeyboard('Keyboard_us','eng',true);
-    keymanweb.doKeyboardChange('Keyboard_us','eng',true);
+    keymanweb.setActiveKeyboard(''); // This line is used in all the UI elements to set the default keyboard, so we should follow suit here.
```

Comment:
`setDfltKeyboard` is a private function, right? Not a published API? AFAICT, it's used only when a keyboard fails to load and the alert is shown (kmwnative.js:104). So I'm a little unsure what the change in behaviour here is supposed to achieve. The original doesn't seem right, but neither does the change to turn KeymanWeb off entirely?